src/CheckOut.js

```diff
@@ -547,11 +554,13 @@ class CheckOut extends React.Component {
             <SafeHTMLMessage
               id="ui-checkout.awaitingPickupMessage"
               values={{ count: requestsCount }}
-            />}
+            />
+          }
```

Comment:
Please be consistent with braces. I see the different usage [here](https://github.com/folio-org/ui-checkout/pull/441/files#diff-cf530b9f5da48ba24dbabbd8f32a36f4R542). Please make all similar consistent.

Reply:
The curly braces can not be placed similar consistent. I placed it up to lint settings.
It requires to :

-  replace closing brace to the next line if expression relates to attribute

- leave  closing brace at the same line if this is embedding expression in jsx